input/pagecontent/reference.md

```diff
@@ -1653,6 +1650,70 @@ A filter is a logical statement in the form `{property} {operator} {value}`, whe
 
 > **Note:** Intensional and extensional forms can be used together in a single value set definition.
 
+{%include tu-div.html%}
+
+##### Concept Metadata
+
+Within a ValueSet definition, the caret syntax MAY be used to set metadata attributes for individual concepts (e.g., elements of ValueSet.compose.include.concept.designation).
+
+To assign metadata values for concepts that are included in the value set, authors SHOULD specify one or more indented caret path assignment rules below the rule that includes the concept:
```

Comment:
Same as above for "SHOULD".  Change to "authors can specify"

Reply:
We had indicated that we had a strong preference for indent syntax in this case -- so I was using SHOULD in the sense that it is RECOMMENDED. We don't want to REQUIRE people use this syntax, but we _almost_ do -- because it's a lot more clear, especially when you get into the exclude rules.

Would you like me to take out the formal SHOULD and add a line at the end that says
> This is the recommended approach:

Reply:
Or - we can document only the indented syntax and leave the other out. (if the latter happens to work in SUSHI, that's a bonus). I think that is my preference. Probably the only case of requiring indentation, but I think it will be ok unless you have strong objections.

Reply:
I don't know. Officially supporting the un-indented form mirrors how you can do it in CodeSystems. And it's so similar to the feature in code systems that I think not supporting it could be confusing. I also don't think I'd want to exclude it from the spec but include it in SUSHI because then we are encouraging people to write non-spec-compliant FSH. And if we only supported indents, then we'd need to change other parts of the spec that explicitly say that indenting is always optional.

So I think we should keep both forms but encourage indenting.

Reply:
I just changed the language in d48596aa772190b3c223caa1883b4e974f3f970f, but kept the repeated code approach. If you feel really strongly about you can negative comment it. ;-). Or we can take it out and then I can negative comment it.